data/colors/themes/light.ts

```diff
@@ -72,4 +72,4 @@ const exceptions = {
   }
 }
 
-export default merge(lightDeprecatedVars, lightAppVars, lightMarketingVars, lightComponentVars, lightGlobalVars, {scale})
```

Comment:
These light mode "exceptions" will actually propagate to all other light themes because they all extend this light theme. I think we should avoid defining `exceptions` in the base `light` and `dark` themes.

Reply:
Ohh.. mean they are not really "exceptions" and more like "overrides" that will be inherited in all the other themes?

Hmm.. ok. I'll add the hex values for the primary button back to the `light` theme. Hopefully we can reduce these hex values so we don't exceptions for the rest of the themes.

/cc @Juliusschaeper

Reply:
Added [a note](https://github.com/primer/primitives/pull/246/files#diff-00d50f3b99c855ce06d9098f82ed4bd2eef4a3ea42b655d99eab9ea986edcb7dR66-R69) ☝️  about not adding exceptions since this will probably come up at some point again.